apps/sync/index.html

```diff
@@ -19,4 +19,5 @@
   <script src="shared/js/lazy_loader.js"></script>
   <script src="shared/js/iac_handler.js"></script>
   <script src="shared/js/sync/bootstrap.js"></script>
+  <script src="js/scratchpad.js"></script>
```

Comment:
This is only for debugging purpose. Sorry for the misunderstanding.
Please see the commit which will be landed to master:
[https://github.com/weilonge/gaia/commit/13a9b6d87f08f366f8ed7325b69d0bfd28efe406]